ansible/localhost-setup-ubuntu20.04-devpc.yml

```
@@ -17,5 +17,5 @@
     - { role: tensorrt, when: yn_gpu == 'y' }
     - { role: ros2, rosdistro: foxy }
     - { role: autoware, rosdistro: foxy }
-    # - { role: pacmod, rosdistro: foxy }
+    - { role: pacmod, rosdistro: foxy }
```

This should be removed after ros2-porting is done.

```suggestion
    - { role: pacmod, rosdistro: foxy }  # Temporary for ros2 porting
```